app/core/context.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {useContext, ReactElement} from "react"
+import {Provider} from "react-redux"
+import BrimApi from "src/js/api"
+import AppErrorBoundary from "src/js/components/AppErrorBoundary"
+import {Store} from "src/js/state/types"
+import theme from "src/js/style-theme"
+import {ThemeProvider} from "styled-components"
+
+const BrimApiContext = React.createContext<BrimApi | null>(null)
```

Comment:
Ah cool, I like this! I've actually been looking for an opportunity to use the context api more in our app and this seems like the perfect use case.

Reply:
Sweet!